src/NumericalAlgorithms/SphericalHarmonics/StrahlkorperFunctions.hpp

```diff
@@ -348,4 +348,25 @@ template <typename Fr>
 void normal_one_form(const gsl::not_null<tnsr::i<DataVector, 3, Fr>*> one_form,
                      const tnsr::i<DataVector, 3, Fr>& dx_radius,
                      const tnsr::i<DataVector, 3, Fr>& r_hat);
+
+/// @{
+/*!
+ * The linear least squares fit of the polynomial of order 3
+ * given a std::vector of Strahlkorpers to the ylm coefficients.
```

Comment:
ticks around `std::vector` and probably do so around `Strahlkorper` as well so it renders as `Strahlkorper`s

Same with a lof of the other things in the dox

Reply:
hmm this renders fine for me without ticks...

Reply:
I mean, it renders, but we typically put ticks around these things in our documentation :) Same with the math below, should probably be latexed. E.g. `\f$l_\max\f$` and `\f$2l_\max m_\max\f$` instead of the `*` in the dox.